Foundation/NSSwiftRuntime.swift

```diff
@@ -81,12 +81,12 @@ internal func _CFZeroUnsafeIvars<T>(_ arg: inout T) {
 }
 
 @_cdecl("__CFSwiftGetBaseClass")
-internal func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
+public func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
     return unsafeBitCast(__NSCFType.self, to:UnsafeRawPointer.self)
 }
 
 @_cdecl("__CFInitializeSwift")
-internal func __CFInitializeSwift() {
+public func __CFInitializeSwift() {
```

Comment:
So I guessed and just selected the test bundle as the profile target for instruments - 

the linker invocation is wrong btw I think... It probably is attempting to link the real CoreFoundation instead of the libCoreFoundation.a